source/administration/command-line-tools.rst

```diff
@@ -989,6 +992,29 @@ mattermost import slack
     .. code-block:: none
 
       ./mattermost import slack myteam slack_export.zip
+      
+mattermost integrity
+--------------------
+
+  Description
+    Command to check database schema integrity as well as referential integrity of channels, command web hooks, posts, schemes, sessions, users, and teams. This process may temporarily affect live system performance, and should be used during off-peak periods. 
+    
+  Format
+    .. code-block:: none
+
+      mattermost integrity {flag} {flag}
```

Comment:
Looks like in other commands we don't include the flag, and would just be ``mattermost integrity``

Reply:
^cc quick reminder on this note @justinegeffen :)

Reply:
Thanks for the reminder, I had completely missed it. Update has been made as requested.